src/styles/partials/template/_sidebar.scss

```diff
@@ -16,9 +17,19 @@
     border: none;
     letter-spacing: 10px;
     font-weight: $font-bold;
+    font-size: $h3-font-size;
   }
 
   p {
     line-height: 30px;
   }
 }
+
+@media screen and (max-width: $medium-breakpoint) {
+  .sidebar {
```

Comment:
I think i maybe confused you here, the border should be at the top of each sidebar section

Reply:
@tmccoy529 we alsoo probably want to create a class for this.

Reply:
this section was actually reworked and has changed a bit.